notebooks_community/FuRBO/FuRBO.ipynb

```diff
@@ -0,0 +1,1011 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b5831947-283e-4682-aae4-bd19bcce03e0",
+   "metadata": {},
+   "source": [
+    "# Feasibility-driven trust Region Bayesian Optimization (FuRBO)\n",
+    "\n",
+    "- Contributors: paoloascia, elenaraponi\n",
+    "- Last update 17 October 2025\n",
+    "- BoTorch version: 0.12.0\n",
+    "\n",
+    "This tutorial shows how to implement Feasibility-driven trust Region Bayesian Optimization (FuRBO) with restarts in a closed loop [1].\n",
```

Comment:
In general, we strive to explain the method, how and why it works to users in the tutorials. In this regard, the notebook is currently lacking a bit. I think an overarching goal should be to have a potential user understand when the method should be used, and ideally see the intuition behind it. Here is one example where there are both visualizations and some explanations of the method accompanying the implementation:


https://github.com/meta-pytorch/botorch/blob/main/tutorials/relevance_pursuit_robust_regression/relevance_pursuit_robust_regression.ipynb

This is admittedly an extreme example, but hopefully it demonstrates the point. As an example, what does FurBO accomplish that TurBO/SCBO does not in this particular case?